### PR TITLE
EndPoints MVP completed

### PR DESCRIPTION
**Pull request by irvinmoore**

### CODE REVIEW

File: app/controllers/cats_controller.rb [hunk 1]
```diff
@@ -1,3 +1,25 @@
...
+      private
+      def cat_params
+        params.require(:cat).permit(:name, :age, :enjoys, :image)
+      end
+  
+    def update
+    end
+  
+    def destroy
+    end
```
Comment (sjproctor) [inline app/controllers/cats_controller.rb:22]: Please fix your indentation on this file please.

**Review by sjproctor — COMMENTED** [review]
just fix the indentation please!

**Review by sjproctor — APPROVED** [review]

**Review by sjproctor — CHANGES_REQUESTED** [review]
Are your tests passing?

**Review by sjproctor — APPROVED** [review]

**Comment by irvinmoore** [comment]
Thank you.

**Pull request merged**